packages/components/src/config/types.ts

```diff
@@ -0,0 +1,3 @@
+export type UIVariant = 'primary' | 'secondary' | 'tertiary' | 'info' | 'warning' | 'destructive';
```

Comment:
I like the idea of shared "Variants".  I have a heretic idea to rename it to `Intent` and remove/rename `'secondary' | 'tertiary'` to some intent-like naming.

What is the communicated intent when you are using them?

Reply:
This probably needs to be discussed with design team, but TBH I have no idea when to use `secondary`. But it is on the other hand very clear when to use `info` or `destructive` (or more mainstream naming: `danger`)